contrib/win32/openssh/OpenSSHBuildHelper.psm1

```diff
@@ -542,18 +542,22 @@ function UnInstall-OpenSSH
         [string]$OpenSSHDir = "$env:SystemDrive\OpenSSH"
     )
 
+    if (-not (Test-Path $OpenSSHDir))
```

Comment:
this check is needed. if the folder does not exists, there is no need to do any of the below steps. The purpose of stopping ssh-agent is to run uninstall script. If the folder does not exists, there is no need to stop ssh-agent either.